### ISSUE TITLE
Still maintained ?

### ISSUE THREAD

**Issue by fgmehlin** [ahopkins/sanic-jwt#247]
Hi there, 

I'm starting to use this cool library in a project and would like to see if this is still being actively supported. 

I'd be happy to consider contributing as well if needed.

**Comment by ahopkins** [comment]
Yes. I badly need to get a new release out. I'm hoping to get that done soon. 

But, if you are interested, I'm absolutely happy to have your help!

**Comment by fgmehlin** [comment]
Cool, very glad to hear that :). 

Do you have anything on the smaller size I could get started with? It's a bit of a busier time on my end but I'd love to help if I can.

E.g. https://github.com/ahopkins/sanic-jwt/issues/167 or would anything else help ?

I could also help type the codebase.

**Comment by ahopkins** [comment]
TBH, I have been kicking around in my head a v2. Trying to keep the top-level API the same, but really update the whole codebase.

**Comment by Pirulax** [comment]
Yes it pretty much seems like nobody is maintaining it anymore..
24.6 came out and now this library basically doesn't even work because of #254.
I even submitted a fix #255...
